### DISCUSSION TITLE
Cypress tests to self-hosted solutions will require the Cypress Gateway Connector

### DISCUSSION THREAD

**Discussion by splincode** [sorry-cypress/cy2#79] [Announcements]
### Description

<img width="1304" alt="image" src="https://github.com/sorry-cypress/cy2/assets/12021443/f6b72ab2-dfcf-4f85-9422-dcbb5cdae525">

### Example usage

https://go.cypress.io/continue-recording-with-gateway-connector?id=eHR3d34yaH51d2p4eA%3D%3D

**Comment by agoldis** [comment]
Please use https://github.com/currents-dev/cypress-cloud instead
